integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -28,29 +28,17 @@
 
 pytestmark = pytest.mark.nightly_resource_consuming_test
 
-_no_nans_float_conf = {'spark.rapids.sql.variableFloatAgg.enabled': 'true',
-                       'spark.rapids.sql.hasNans': 'false',
-                       'spark.rapids.sql.castStringToFloat.enabled': 'true'
-                      }
+_float_conf = {'spark.rapids.sql.variableFloatAgg.enabled': 'true',
+                'spark.rapids.sql.castStringToFloat.enabled': 'true'
+                }
```

Comment:
Nit: Indentation is off here.
```suggestion
               'spark.rapids.sql.castStringToFloat.enabled': 'true'
              }
```